pkg/local-storage/member/controller/volume_migrate_task_worker.go

```diff
@@ -29,12 +29,14 @@ func (m *manager) startVolumeMigrateTaskWorker(stopCh <-chan struct{}) {
 				break
 			}
 			if err := m.processVolumeMigrate(task); err != nil {
+				m.volumeMigrateLimit <- struct{}{}
```

Comment:
this method can't prevent multiple migration tasks from being carried out at the same time, because each time a task fails, the "token" will be returned.